docs/Developers/Review.rst

```diff
@@ -15,7 +15,7 @@ on your own.
 
 You can totally help us review source code. Especially try to review source
 code of others and share what you have learnt with them. You can use acks and
-unacks like everyone else and ``cobot`` even allows you to set PRs to WIP. Check
+unacks like everyone else and ``corobot`` even allows you to set PRs to WIP. Check
```

Comment:
It should be `corobo` not `corobot`.

Reply:
Also, move some words in this line to next line to limit line length to 80

Reply:
Thankyou @Vamshi99 for your review